transform/transformers/common_software/pck_transformer.py

```diff
@@ -301,8 +456,10 @@ def derive_answers(self):
         except KeyError:
             logger.info("Missing metadata")
 
+        # Important: Round first, then calculate totals, otherwise the totals won't add up correctly
```

Comment:
Has this been confirmed with the business ? It looks better when they tally , but I dont know the business use of the total.

Reply:
Fair point, I'll mention it to them and see what's right